README.md

```diff
@@ -252,6 +261,10 @@ custom:
             input:
               key-a: "value-a"
               key-b: "value-b"
+
+      # Deprecated: number of simultaneous requests to handle
+      # Please use scalingOption of type concurrentRequests instead
+      maxConcurrency: 20
```

Comment:
suggestion: since it's deprecated, maybe we can keep it but commented
Also, if someone copy/paste blindly as is, it would throw an error because both `maxConcurrency` and `scalingOption` are specified 
```suggestion
      # maxConcurrency: 20
```

Reply:
great idea, done :white_check_mark: